tree/30_generic_methods/package_install_version_cmp_update.cf

```diff
@@ -141,7 +141,7 @@ bundle agent package_install_version_cmp_update(package_name, version_comparator
                         package_policy  => "${update_policy}",
                         package_version => "${package_version}",
                         package_select  => "${version_comparator}",
-                        package_method  => ncf_generic,
+                        package_method  => ncf_generic_version,
```

Comment:
Could you use ncf_generic_version for previous case also, so that we have a consistent implementation